modules/apps/object/object-test/src/testFunctional/tests/ObjectFields.testcase

```diff
@@ -2720,6 +2720,75 @@ definition {
 		}
 	}
 
+	@description = "Verify that the Is Equals to operator is available when adding filters for the long integer fields to select data from the object"
+	@priority = "4"
+	test IsEqualToIsAvailableForLongIntegerFields {
+		task ("Given: A custom object with a long integer field type") {
+			ObjectAdmin.addObjectViaAPI(
+				labelName = "Custom Object",
+				objectName = "CustomObject",
+				pluralLabelName = "Custom Objects");
+
+			ObjectAdmin.addObjectRelationshipViaAPI(
+				objectName_1 = "CustomObject",
+				objectName_2 = "CustomObject",
+				relationshipLabel = "Relationship",
+				relationshipName = "relationship",
+				relationshipType = "oneToMany");
+
+			ObjectAdmin.openObjectAdmin();
+
+			ObjectPortlet.selectCustomObject(label = "Custom Object");
+
+			ObjectAdmin.goToFieldsTab();
+
+			ObjectAdmin.addObjectFieldViaUI(
+				fieldLabel = "Custom Long Integer",
+				fieldName = "customLongInteger",
+				fieldType = "Long Integer");
+
+			ObjectAdmin.addObjectFieldViaUI(
+				field = "Custom Long Integer",
+				fieldLabel = "Custom Aggregation",
+				fieldType = "Aggregation",
+				function = "Max",
+				relationship = "Relationship");
+		}
+
+		task ("When: A new filter is created with the custom long integer field with Is Equal To selected") {
+			ObjectAdmin.goToFieldsDetails(label = "Custom Aggregation");
+
+			Click(locator1 = "ObjectCustomViews#ADD_BUTTON_FILTERS");
+
+			Click(locator1 = "ObjectCustomViews#VIEW_MODAL_CONTENT");
+
+			Click(
+				key_picklistOption = "Custom Long Integer",
+				locator1 = "ObjectCustomViews#FIELD_VIEW_FILTER_DROPDOWN_OPTION");
+
+			Click(
+				key_filterLabel = "Filter Type",
+				locator1 = "ObjectCustomViews#SELECT_FILTER_GENERIC");
+
+			Click(
+				key_fieldType = "Is Equal To",
+				locator1 = "ObjectField#TYPE_OPTION");
+
+			Type(
+				key_text = "Value",
+				locator1 = "TextInput#ANY",
+				value1 = "1");
+
+			ObjectCustomViews.clickSaveOnModal();
+		}
```

Comment:
This is fine for now but we should create a separate macro when creating the Custom Aggregation field.